src/components/i18n/FormattedCompMessage.js

```diff
@@ -1,4 +1,5 @@
 // @flow
+// @deprecated, use FormattedMessage from react-intl v6 instead.
```

Comment:
Can we delete these components at this point since the upgrade itself is already a breaking change?